### ISSUE TITLE
Feature request: Export SVG / XML

### ISSUE THREAD

**Issue by heylix** [tchayen/markdown-links#52]
Would be nice if you could export the generated link graph like [edeno/d3-save-svg](https://github.com/edeno/d3-save-svg) does.

**Comment by tchayen** [comment]
Agree, it would be good to have an option to export both pure graph data as a JSON/XML and the image representation in SVG. I am not sure whether I will be able to work on it any time soon, but any PRs with such work are very welcome.

(actually, it is even easier now than it will be in the future since extension currently uses SVG as a rendering layer but there is ongoing work to migrate it to canvas/WebGL; nothing relevant, just mentioning it for the record)

**Comment by deconDarts** [comment]
Clickable embed for UX/UI Nav on live site.
